RockLib.Messaging.Http/RequiredHttpRequestHeaders.cs

```diff
@@ -46,48 +47,62 @@ public RequiredHttpRequestHeaders(string contentType = null, string accept = nul
         /// response. When this value is null, HTTP requests are not filtered according to their
         /// <c>Content-Type</c> header.
         /// </summary>
-        public string ContentType { get; }
+        public string? ContentType { get; }
 
         /// <summary>
         /// Gets the <c>Accept</c> header value(s) that incoming HTTP requests must match in order to
         /// be handled. When specified, any request that does not have a <c>Accept</c> header that
         /// matches this value will immediately receive a <c>406 Not Acceptable</c> response. When this
         /// value is null, HTTP requests are not filtered according to their <c>Accept</c> header.
         /// </summary>
-        public string Accept { get; }
+        public string? Accept { get; }
 
-        internal bool AllowsContentType(string requestContentType)
+        internal bool AllowsContentType(string? requestContentType)
         {
-            if (_contentTypeMediaTypes == null)
+            if (_contentTypeMediaTypes is null)
+            {
                 return true;
+            }
 
             try
             {
                 var contentType = MediaTypeHeaderValue.Parse(requestContentType);
                 return _contentTypeMediaTypes.Any(mediaType => contentType.MediaType == mediaType);
             }
+#pragma warning disable CA1031 // Do not catch general exception types
```

Comment:
Is there a reason we couldn't remove this exception block? Maybe the `Parse()` could be changed into a `TryParse()`? The `Any()` really shouldn't fail for a `MediaType` comparison.

Reply:
I updated this to `TryParse()`.